### PR TITLE
feat(cardinal): allow cors on dev mode

### PR DESCRIPTION
**Pull request by pyrofolium**
Closes: #500

## What is the purpose of the change

Allow CORS as an option for servers. 

## Testing and Verifying

Added an additional check to an existing test

### CODE REVIEW

File: cardinal/option.go [hunk 1]
```diff
@@ -82,3 +82,9 @@ func WithPrettyLog() WorldOption {
 		ecsOption: ecs.WithPrettyLog(),
 	}
 }
+
+func WithCORS() WorldOption {
+	return WorldOption{
+		serverOption: nil,
+	}
+}
```
Comment (technicallyty) [inline cardinal/option.go:90]: something missing here 🕵🏻‍♂️

Comment (pyrofolium) [reply cardinal/option.go:90]: eh good catch.

Comment (pyrofolium) [reply cardinal/option.go]: fixed added coverage for this function too.

**Review by technicallyty — APPROVED** [review]
LGTM

**Pull request merged**